src/sidebar/index.js

```diff
@@ -301,6 +311,7 @@ function startAngularApp(config) {
     .service('analytics', () => container.get('analytics'))
     .service('annotationMapper', () => container.get('annotationMapper'))
     .service('annotationsService', () => container.get('annotationsService'))
+    .service('autosaveService', () => container.get('autosaveService'))
```

Comment:
Can you keep the `import <service>`,  `.register(<name>, service)` and `.service(<name>, container.get(...))` calls sorted in alphabetical order please?

Reply:
Absolutely. Thanks for keeping me honest here!